utils/mux_gen.py

```diff
@@ -250,13 +250,13 @@ def normpath(p, to=None):
             if args.split_inputs:
                 port_names.extend((mux_lib.MuxPinType.INPUT, args.name_inputs[j], 1, '[%i]' % j) for j in range(args.width))
             else:
-                port_names.append((mux_lib.MuxPinType.INPUT, args.name_input, args.width, '[%i:0]' % args.width))
+                port_names.append((mux_lib.MuxPinType.INPUT, args.name_input, args.width, '[%i:0]' % (args.width-1)))
```

Comment:
Can you add a comment here? I think it should be something like "Verilog indexes are inclusive"?